src/ape/cli/options.py

```diff
@@ -597,3 +597,28 @@ def _json_option(name, help, **kwargs):
 
 def config_override_option(**kwargs):
     return _json_option("--config-override", help="Config override mappings", **kwargs)
+
+
+def _excluded_compilers_callback(ctx, param, value):
+    if not value:
+        return
+
+    return [c.lower() for c in value]
+
+
+def excluded_compilers_option(**kwargs):
+    from ape.utils.basemodel import ManagerAccessMixin
+
+    registered_compilers_options = list(
+        ManagerAccessMixin.compiler_manager.registered_compilers
+    )
```

Comment:
I have to stop editing from github lmao
```suggestion
    registered_compilers_options = list(ManagerAccessMixin.compiler_manager.registered_compilers)
```